web/modules/custom/pineapple_recipe/pineapple_general.module

```diff
@@ -29,13 +29,19 @@ function pineapple_recipe_help($route_name, RouteMatchInterface $route_match) {
 /**
  * Implements hook_form_alter().
  */
-function pineapple_recipe_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
+function pineapple_form_alter(&$form, \Drupal\Core\Form\FormStateInterface $form_state, $form_id) {
```

Comment:
This should be `pineapple_general_form_alter`

Reply:
core.extension file, pineapple_recipe file and their contents should be updated to have the module renamed to pineapple_general

Reply:
And references to recipe in this file should be changed to general

Reply:
done